sources/repo-daemon

```diff
@@ -36,6 +36,7 @@ func_help() {
                               '/etc/repo-scripts.conf'"
   echo -e "\n  -v   --version            - show version and exit"
   echo -e "  -h   --help               - show this help and exit"
+  echo -e " --no-daemon               - Execute one shot"
```

Comment:
just a simple question... What is real use case for this option? As far as I understand it may be only useful if you want to build OOD packages and update html once, am I right?